base/services/umpnpmgr/rpcserver.c

```diff
@@ -2611,7 +2617,7 @@ PNP_GetInterfaceDeviceListSize(
         ret = NtStatusToCrError(Status);
     }
 
-    DPRINT("PNP_GetInterfaceDeviceListSize() done (returns %lx)\n", ret);
+    DPRINT("PNP_GetInterfaceDeviceListSize() done (listSize %lx,returns %lx)\n",*pulLen , ret);
```

Comment:
Not sure it's important to log the listsize for final patch... (especially since *pulLen is undefined in case the function returned an error).

Reply:
in case of error we will see the error code and pulLen not changed
but in case of success we will see the returned value, and this is extremally useful